lib/constants.cjs

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+Object.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });
```

Comment:
question

Can you add a comment that explains why it's needed?
Or is it a practice that we have introduced in another PR? In that case which one?
I am more familiar with
```js
if (typeof Symbol !== 'undefined' && Symbol.toStringTag) {
  Object.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });
}
Object.defineProperty(exports, '__esModule', { value: true });
```

Reply:
This is an auto-generated code by Rollup. And [`Symbol` has been supported since older Node.js versions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol#browser_compatibility), so I think extra checking is unnecessary.